### PR TITLE
SECO-3082: halow basic functionality test cases

### PR DESCRIPTION
**Pull request by murugesan-kumar**
halow (nrc7292) functionality test

### CODE REVIEW

**Review by DamienDeMayaTII — APPROVED** [review]
Looks ok to me

**Comment by joenpera** [comment]
This will work with USB connected Halow card and not with the direct SPI one.

**Review by joenpera — APPROVED** [review]

File: common/tests/functional/halow/test_halow_nrc7292.sh [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+# Globals:
+# Arguments:
+#######################################
+get_country() {
+  country_str=$(cli_app show config | grep "Country")
+
+  if [ -z "$country_str" ]; then
+    echo "Failed to retrieve country information"
+  else
+    # Parse and retrieve mcs value
```
Comment (mbaddeley) [inline common/tests/functional/halow/test_halow_nrc7292.sh:85]: Incorrect comment?

**Comment by murugesan-kumar** [comment]
> This will work with USB connected Halow card and not with the direct SPI one.

Yes, Got it Mika. Once we receive CCB3.0 which uses Silex NEWAH, I will update the script to support the same.

File: common/tests/functional/halow/test_halow_nrc7292.sh (hunk 1 above)
Comment (murugesan-kumar) [reply common/tests/functional/halow/test_halow_nrc7292.sh:85]: yes.. it is typo!

**Review by ZAbidTii — APPROVED** [review]

**Pull request merged**